index.html

```diff
@@ -248,6 +250,14 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="i">I</div>
 
+<p><dfn id="def_ijam" class="lint-ignore export">Ijam</dfn>. An ijam is an Arabic script diacritical mark considered to be part of a basic letter form. &#x681; [<span class="uname" translate="no">U+0681 ARABIC LETTER HAH WITH HAMZA ABOVE</span>] is an example of a letter with an ijam. Unicode encodes letter+ijam combinations as separate, atomic characters, which are never given decompositions in the standard. Ijam generally take the form of one-, two-, three- or four-dot markings above or below the basic letter skeleton, although other diacritic forms occur in extensions of the Arabic script in Central and South Asia and in Africa.</p>
+
+<aside class="note">
+	<p>This distinction between using a character with <a>ijam</a> instead of combining a letter with a <a>tashkil</a> becomes important when choosing which Unicode characters to use because (as can be seen in the examples above) the visual forms can be identical. Using the wrong character can change the meaning of the text, affecting the results of text search, font rendering, text to speech, etc.</p>
```

Comment:
I would like us to remove this paragraph and the following one.
Same applies to the tashkil section.